etc/ATSCam/primary_hdu.header

```diff
@@ -30,12 +30,12 @@ OBSGEO-Y=          -5208470.95 / [m] Y-axis Geocentric coordinate
 OBSGEO-Z=          -3195172.08 / [m] Z-axis Geocentric coordinate
 
         ---- Pointing info, etc. ----
-RA      =                      / Telescope RA 
-DEC     =                      / Telescope DEC
-RASTART =                      / RA of Target at start
-DECSTART=                      / DEC of Target at start
-RAEND   =                      / RA of Target at end
-DECEND  =                      / DEC of Target at end
+RA      =                      / Telescope RA from pointing component
```

Comment:
RA commanded from pointing component
--> It's important to know it's the commanded position, not necessarily the actual position. No need to say "telescope"

Reply:
Changes to `"RA commanded from pointing component"` and `"DEC commanded from pointing component"`